src/main/java/core/basesyntax/strategy/filestrategy/ReportBuilderHandler.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.strategy.filestrategy;
+
+import core.basesyntax.strategy.filestrategy.impl.CsvReportBuilderImpl;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+public class ReportBuilderHandler {
+    private final Map<String, ReportBuilder> reportBuilders = new HashMap<>();
```

Comment:
```suggestion
    private final Map<String, ReportBuilder> reportBuildersMap;
```
initialize using constructor